webview/package.json

```diff
@@ -7,6 +7,7 @@
     "lint:prettier": "prettier -c '**/*.{js,json,jsx,md,scss,ts,tsx,yaml,yml}'",
     "lint:eslint": "eslint --cache '**/*.{js,ts,jsx,tsx}'",
     "lint:tsc": "tsc --noEmit",
+    "lint:webpack": "webpack --mode development",
```

Comment:
What is the use of this new task?

Reply:
Forgot to mention it in the description.

The first time I moved the files VSCode didn't automatically fix the styles.scss imports, and the error wasn't caught by the commit hook because it relied only on the typescript compiler which was ok with it.

So I decided to add this to the linting tasks to catch errors like those locally.

Reply:
Cool. Can you link it to the root of the project (add a task in the root package.json that calls this task)? It'll be easier to reuse it that way if we don't have to `cd` into the webview folder.

Reply:
Please move this unrelated change out, we need to be able to come back and find the reason for the change from the PR.

Reply:
Will do, thanks!